packages/app/src/components/dialog-select-directory.tsx

```
@@ -21,157 +21,131 @@ type Row = {
-  )
-
-  const home = createMemo(() => sync.data.path.home || fallbackPath()?.home || "")
-
-  const start = createMemo(
-    () => sync.data.path.home || sync.data.path.directory || fallbackPath()?.home || fallbackPath()?.directory,
-  )
-
-  const cache = new Map<string, Promise<Array<{ name: string; absolute: string }>>>()
+function cleanInput(value: string) {
+  const first = (value ?? "").split(/\r?\n/)[0] ?? ""
+  return first.replace(/[\u0000-\u001F\u007F]/g, "").trim()
```

Biome (2.3.14)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)